addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2216,13 +2217,15 @@ options.registry.topMenuColor = options.Class.extend({
     /**
      * @override
      */
-    selectStyle(previewMode, widgetValue, params) {
+    async selectStyle(previewMode, widgetValue, params) {
         this._super(...arguments);
         const className = widgetValue ? (params.colorPrefix + widgetValue) : '';
-        this.trigger_up('action_demand', {
+        await new Promise((resolve, reject) => this.trigger_up('action_demand', {
```

Comment:
Should already have been the case (but not strictly needed in that case I guess), but I would await the call to `_super` too. Although I am curious: why what the action_demand call not async before? Error in stable or unneeded async stuff here? :thinking:

Reply:
14.0 PR is here : https://github.com/odoo/odoo/pull/95896